### PR TITLE
`arm01`: Postsubmit that triggers a ci-operator-arm64 BuildConfig

### PR DESCRIPTION
**Pull request by danilo-gemoli**
A postsubmit that triggers the BuildConfig `ci-operator-arm64` every time something gets merged on [o/ci-tools](https://github.com/openshift/ci-tools).
The arguments `--follow` and `--wait` make sure that the ProwJob streams the build log in realtime and waits until it (the build) either fails or succeeds.
The `config-updater` SA has rights to trigger any BuildConfig from the `ci` NS.

Jira Card: [DPTP-3591](https://issues.redhat.com/browse/DPTP-3591)

/cc @droslean @deepsm007

### CODE REVIEW

File: ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml [hunk 1]
```diff
@@ -4,7 +4,44 @@ postsubmits:
...
+          name: build-farm-credentials
+          readOnly: true
+      serviceAccountName: config-updater
+      volumes:
+      - name: build-farm-credentials
+        secret:
+          items:
+          - key: sa.config-updater.arm01.config
+            path: kubeconfig
+          secretName: config-updater
```
Comment (droslean) [inline ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml:39]: Are you sure that this is the correct credentials to trigger the build? I am not sure if the `config-updater` has permission on buildConfigs.

Comment (danilo-gemoli) [reply ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml:39]: Yeah it's the account I'm using to do some tests before opening the PRs as mine doesn't have any power there. Give it a shoot eventually, so to have a double check.

Comment (droslean) [reply ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml:40]: You need to use the `origin-release-images-builder` service account.

Comment (danilo-gemoli) [reply ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml:40]: I agree, but shouldn't then we
- create `origin-release-images-builder` on `arm01` as well as it exists on `build01` only (check [here](https://github.com/openshift/release/blob/dc969377f33de08e69553cee297c0575b592daa7/clusters/build-clusters/01_cluster/ci/admin_origin-release-images-builder_rbac.yaml))?
- and then force the postsubmit of this PR to always run on `arm01` with the SA set to `origin-release-images-builder` as you're suggesting?

File: ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+    spec:
+      containers:
+      - args:
+        - --namespace=ci
+        - start-build
+        - --follow
+        - buildconfig/ci-operator-arm64
+        command:
+        - oc
+        image: image-registry.openshift-image-registry.svc:5000/ocp-arm64/4.13:cli
```
Comment (droslean) [inline ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml:22]: How can we be sure that this job will run on an arm node?

Comment (danilo-gemoli) [reply ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml:22]:
```yaml
cluster: arm01
...
labels:
  ci-operator.openshift.io/cluster: arm01
```
will ensure Prow is going to spawn the pod on the arm cluster.
`image-registry.openshift-image-registry.svc:5000/ocp-arm64/4.13:cli` holds `oc` version `arm64` which will authenticate to the cluster as `ci-images-builder`.

Comment (droslean) [reply ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml:22]: We have both amd64 and arm64 nodes in the arm01 cluster. We need to make sure that the pod will be spawned in the arm64 node.

**Comment by droslean** [comment]
Let's merge it and deal with my comment in a follow-up

/lgtm
/approve

**Pull request merged**